stock_no_negative/models/stock_quant.py

```diff
@@ -33,5 +33,5 @@ def check_negative_qty(self):
                     "stock level of the product '%s'%s would become negative "
                     "(%s) on the stock location '%s' and negative stock is "
                     "not allowed for this product.") % (
-                        quant.product_id.name, msg_add, quant.qty,
+                        quant.product_id.name_get()[0][1], msg_add, quant.qty,
```

Comment:
```suggestion
                        quant.product_id.display_name, msg_add, quant.qty,
```

I think this is more correct

Reply:
Done!